CICD/ExtensionMethods.cs

```diff
@@ -252,6 +267,50 @@ public static BranchType GetBranchType(this string branch)
         return BranchType.Other;
     }
 
+    /// <summary>
+    /// Converts the given path string to a cross-platform compatible path.
+    /// </summary>
+    /// <param name="value">The <c>string</c> path to convert.</param>
+    /// <returns>A cross-platform path.</returns>
+    /// <remarks>
+    /// <para>
+    ///     This simply takes all windows specific directory separator characters and
```

Comment:
Refactor to remove simply
```suggestion
    ///     This takes all windows specific directory separator characters and
```